src/domain/collaboration/whiteboard/WhiteboardsManagement/WhiteboardManagementView.tsx

```diff
@@ -104,7 +104,9 @@ const WhiteboardManagementView: FC<WhiteboardManagementViewProps> = ({
           <WhiteboardDialog
             entities={{
               whiteboard: entities.whiteboard as WhiteboardValueFragment & WhiteboardDetailsFragment,
-              lockedBy: lockedByDetailsData?.users[0],
+              lockedBy: isWhiteboardAvailable
+                ? undefined
+                : lockedByDetailsData?.users?.find(user => user.id === whiteboard?.checkout?.lockedBy),
```

Comment:
useWhiteboardLockedByDetailsQuery is returning many users, not only the one queried. that sounds like a server bug:
![image](https://github.com/alkem-io/client-web/assets/16032487/611a2f09-4cd5-4a44-b64b-c9f974b8a7c9)
![Uploading image.png…]()